code/game/objects/effects/posters/poster.dm

```diff
@@ -87,7 +91,8 @@
 	balloon_alert(user, "hanging poster...")
 	var/obj/structure/sign/poster/placed_poster = poster_structure || new poster_type(src)
 	placed_poster.poster_item_type = type
-	placed_poster.forceMove(wall_structure)
+	placed_poster.store_sign_in_item(wall_structure)
```

Comment:
What are you doing here exactly? `wall_structure` is of type turf

https://github.com/tgstation/tgstation/blob/c40b1189cc08f1a36b5a9b4fbd19a7ada9303357/code/game/objects/effects/posters/poster.dm#L69
 
this proc is meant to store this poster structure in a poster not in a wall

https://github.com/tgstation/tgstation/blob/c40b1189cc08f1a36b5a9b4fbd19a7ada9303357/code/game/objects/effects/posters/poster.dm#L261-L263

This `stored_ref` will contain a ref to the wall???